include/fiction/algorithms/simulation/sidb/operational_domain.hpp

```diff
@@ -229,6 +275,21 @@ struct operational_domain_value_range
  */
 struct operational_domain_params
 {
+    /**
+     * This enumeration defines whether specific metrics, such as critical temperature,
+     * are simulated within the operational domain.
+     */
+    enum class metric_simulation : uint8_t
```

Comment:
Actually, this enumeration doesn't define such behavior. It simply offers the options to do so. The respective logic is handled elsewhere.